transmit-lib/src/cmr/transmit/config.clj

```diff
@@ -287,3 +287,8 @@
 (defmethod application-public-root-url :default
   [context]
   (format-public-root-url (get-in context [:system :public-conf])))
+
+(defconfig app-environment
+  "The environment in which the application is running in NGAP (wl, sit, uat, ops)"
+  {:default "local"
```

Comment:
question, is the CMR standard to default to production values or development values? This is not safe for production as defined.

Reply:
I actually copied that from the message-queue-lib config, but this is now removed from my PR